src/tests/math/math_test_fp.c

```diff
@@ -0,0 +1,66 @@
+/**
+ * @math_test_fp
+ * @simple tests for fp functions in embox just to make sure the corresponding functions are not stubs
+ * @date 03.03.2020
+ * @author Rushil kvs
+ */
+
+#include <embox/test.h>
+#include <math.h>
+
+EMBOX_TEST_SUITE("Tests for fp functions");
+
+bool equal(double a, double b)
+{
+	double epsilon = 0.00000000000001;
+	if ((a-b) < epsilon && -(a-b) < epsilon) return true;
+  	return false;
+}
+
+TEST_CASE("Test for modf") {
+	double i,t,j;
+	i = modf(2.34, &t);
+	j = 0.34;
+	test_assert_true(equal(i, j));
+	test_assert_true(equal(t, 2.0));
```

Comment:
Wouldn't just the following code work?

```
	i = modf(2.34, &t);
	j = 0.34;
	test_assert_equal(i, j);
	test_assert_equal(t, 2.0);
```

Reply:
Ok, let's use your variant.

Reply:
Here I just meant that we actually do not know the possible error, so my proposition was to choose such numbers to make it just `0.0`. But let's use the `epsilon` everywhere here.